MathGame.destocot/MathGame/Helpers.cs

```diff
@@ -0,0 +1,127 @@
+﻿using MathGame.Models;
+
+namespace MathGame
```

Comment:
Change this to a file-scoped namespace. Remove the associated curly braces and add a semi-colon instead.

```suggestion
namespace MathGame;
```